snippets/sim-periph/sim-periph.conf

```diff
@@ -0,0 +1,9 @@
+# Flash
```

Comment:
is there ever a time we want the old type of sim network interface? if so, would we want to like split this into 2 so theres the FS access snippet and then you pick from one of two NET snippets

Reply:
Not thinking of a reason to at the moment. If there is we can, but I want to try and avoid supporting two separate ways of doing things